HdmiInput/doc/HdmiInputPlugin.md

```diff
@@ -380,17 +416,24 @@ Activates the specified HDMI Input port as the primary video source.
 ```json
 {
     "jsonrpc": "2.0",
-    "id": 1234567890,
+    "id": 42,
     "result": {
         "success": true
     }
 }
 ```
 
 <a name="method.stopHdmiInput"></a>
-## *stopHdmiInput <sup>method</sup>*
+## *stopHdmiInput [<sup>method</sup>](#head.Methods)*
 
 Deactivates the HDMI Input port currently selected as the primary video source.
+ 
+### Events 
+| Event | Description | 
+| :----------- | :----------- | 
+| `onInputStatusChanged` | Triggers the event when deactivates HDMI Input source and Input status changes to `stopped`.
```

Comment:
Change "deactivates HDMI Input source" to "HDMI Input source is deactivated".

Reply:
Updated the changes.